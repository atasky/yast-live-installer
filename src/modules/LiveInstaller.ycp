/****************************************************************************

Copyright (c) 2002 - 2012 Novell, Inc.
All Rights Reserved.

This program is free software; you can redistribute it and/or
modify it under the terms of version 2 of the GNU General Public License as
published by the Free Software Foundation.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program; if not, contact Novell, Inc.

To contact Novell about this file by physical or electronic mail,
you may find current contact information at www.novell.com

****************************************************************************/

/**
 * File:	modules/LiveInstaller.ycp
 * Package:	Configuration of live-installer
 * Summary:	LiveInstaller settings, input and output functions
 * Authors:	Jiri Srain <jsrain@suse.cz>
 *
 * $Id: LiveInstaller.ycp 37292 2007-04-03 16:38:49Z mvidner $
 *
 * Representation of the configuration of live-installer.
 * Input and output routines.
 */

{

module "LiveInstaller";

textdomain "live-installer";

global string live_control_file =
	"/etc/YaST2/live-install.xml";

global boolean run_full_proposal = false;

global list<map<string,any> > source_states = [];

global map<string, map<string,any> > service_states = $[];
/* EOF */
}
