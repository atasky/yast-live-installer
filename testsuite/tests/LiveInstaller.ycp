{

// testedfiles: LiveInstaller.ycp

include "testsuite.ycp";
TESTSUITE_INIT([], nil);

import "LiveInstaller";

DUMP("LiveInstaller::Modified");
TEST(``(LiveInstaller::Modified()), [], nil);

}
