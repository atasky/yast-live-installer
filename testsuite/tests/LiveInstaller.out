Dump	LiveInstaller::Modified
Return	false
